src/iptax/models.py

```
@@ -89,8 +93,8 @@ class ReportConfig(BaseModel):
     @classmethod
     def validate_percentage(cls, v: int) -> int:
         """Validate percentage is in valid range."""
-        if not 0 <= v <= 100:
-            raise ValueError("Creative work percentage must be between 0 and 100")
+        if not 0 <= v <= MAX_PERCENTAGE:
+            raise ValueError(f"Creative work percentage must be between 0 and {MAX_PERCENTAGE}")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)